presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoStatement.java

```diff
@@ -313,45 +372,84 @@ public ResultSet getGeneratedKeys()
     }
 
     @Override
+    public int executeUpdate(String sql)
+            throws SQLException
+    {
+        return Ints.saturatedCast(executeLargeUpdate(sql));
```

Comment:
Same comment as above. The behavior should definitely be consistent across these related methods.